apiserver/facades/client/modelupgrader/upgrader.go

```diff
@@ -307,12 +315,12 @@ func (m *ModelUpgraderAPI) validateModelUpgrade(
 			return errors.Trace(err)
 		}
 		defer st.Release()
```

Comment:
Hi tom, im not 100% sure if this is within the scope of the PR, but would it be the same to get all the UUIDs from ListModelIDS in model service to further reduce the usage of state?
```
ListModelIDs(context.Context) ([]coremodel.UUID, error)
```

Reply:
Thanks,

Yeah we could do that but it doesn't bring us much closer to removing Mongo at the moment. Comment above addresses the plan to fully refactor this facade in the coming weeks and all of this will get removed then.

We can only do small amounts at a time without turning things into very large PR's re-working the world at once.

Reply:
Yup this makes sense as well, thanks tom